tests/models/test_model_yamls.py

```diff
@@ -28,6 +29,11 @@ def walk_model_yamls():
 @pytest.mark.parametrize('hparams_file', walk_model_yamls())
 class TestHparamsCreate:
 
+    @pytest.fixture(scope='function', autouse=True)
+    def ensure_device_cpu(self, monkeypatch: pytest.MonkeyPatch):
```

Comment:
Would prefer not to read `sys.argv` as they can contain pytest-specific args that would conflict with yahp. Instead, would it work to remove the `device` arg from the yaml files? The trainer will automatically use gpu if available; otherwise it will use CPU.

If we need to manually patch the device/precision, could we load the yaml into a dictionary, and then manually override `yaml_dict['device'] = {'cpu': {}}`, and then pass that into `TrainerHparams.create`? The create API supports reading from a dict as well as from a file.

Reply:
Done!